Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py

```diff
@@ -201,6 +205,16 @@ def search_incidents(args: Dict):  # pragma: no cover
         demisto.debug(f"before filtering {len(current_page_found_incidents)=} {args=} {page=}")
         more_pages = len(current_page_found_incidents) == page_size
 
+        # dedup incidents
+        demisto.debug(f"before dedup process with {len(current_page_found_incidents)=}")
+        current_page_found_incidents = [
+            incident for incident in current_page_found_incidents if incident.get("id") not in incidents_ids
```

Comment:
Looking at this, it seems we are claiming that BE might return the same incident twice?

Reply:
Yes indeed, the problem here is an edge case where we call the API twice or more, and in between, new incidents are created... In this case, we request the second page, which is already different as we have new incidents in the system that push the other incident's order...